lib/cdo/share_filtering.rb

```diff
@@ -61,15 +61,15 @@ def self.find_failure(text, locale)
     email = RegexpUtils.find_potential_email(text)
     return ShareFailure.new(FailureType::EMAIL, email) if email
 
-    street_address = Geocoder.find_potential_street_address(text)
-    return ShareFailure.new(FailureType::ADDRESS, street_address) if street_address
-
     phone_number = RegexpUtils.find_potential_phone_number(text)
     return ShareFailure.new(FailureType::PHONE, phone_number) if phone_number
 
     expletive = ProfanityFilter.find_potential_profanity(text, locale)
     return ShareFailure.new(FailureType::PROFANITY, expletive) if expletive
 
+    street_address = Geocoder.find_potential_street_address(text)
+    return ShareFailure.new(FailureType::ADDRESS, street_address) if street_address
```

Comment:
i think part of the plan was to log any address share failures -- should we do that logging as part of this PR, or is that happening separately?

Reply:
Oh sorry, we decided not to do any logging because there would be way too much data to sift through, it's impossible to know how many addresses are passing through the filter, and manual testing has shown that it won't even catch an obvious address right now if it's part of a larger phrase (for example, "123 Main St is where I live" is not registered as an address).